utils/dc-chain/config.mk.stable.sample

```diff
@@ -103,7 +103,7 @@ makejobs=-j2
 # here is to build C, C++, Objective C, and Objective C++. You may want to take
 # out the latter two if you're not worried about them and/or you're short on
 # hard drive space.
-pass2_languages=c,c++,objc,obj-c++
+pass2_languages=c,c++
```

Comment:
Why was this changed?

Reply:
Unintentionally committed, was just during testing while there was issues with the ObjC build. Will revert these unintended changes.